### DISCUSSION TITLE
Triage role seems limited

### DISCUSSION THREAD

**Discussion by michael-hawker** [community/community#5773] [Projects and Issues]
I have triage permissions on a repo, but it seems super limiting to effectively triage issues. The role really makes it feel like the only benefit it adds is the ability to label issues.

However, as someone who triages a lot I really want to be able to also:
- Create new labels
- Transfer issues between repos (that I have at least triage permission to, also see #5766)
- Transfer issues to/from discussions (also see #2971, #3097, #3197, and additional comments on #2861)
- Rename issue titles
- Moderate issues (edit content, hide, report)
- Assign to Projects (not just Milestones)

**Comment by secustor** [comment]
What is also definitely missing are the permissions to transfer an issue to a discussion.   
Vice versa is possible right now.

**Comment by Chew** [comment]
Triage is disappointingly limited. I made a forum post a while ago, detailing a bug and going into more detail about this: https://github.community/t/triage-arent-considered-collaborators/155558

Never got a response, though. My tl;dr boils down to Triage should have the ability to fully manage the issue section, as it's really their job to ensure the only valid issues _and comments_ are constructive for devs (those with write or more) to easily find and resolve the issue.

Another thing I recently ran into is that Triage can't link PRs to Issues or vice versa. If a PR closes an Issue, shouldn't we let GitHub do it? We'd be able to close it manually!

Finally, I was absolutely disappointed to see custom repository roles be enterprise only. Us getting full control over what roles people have should be an everyone thing! I understand the security aspects, and maybe keep those to the enterprise, but the feature itself shouldn't be locked, especially if it's useful for open source projects with volunteer teams of Triage who can really, at most, close issues.

**Comment by Rexogamer** [comment]
This in its entirety. The Triage role already lets a user close issues, so realistically none of the issue features are really that destructive and surely most triagers are trustworthy enough to handle them.

**Comment by chadwhitacre** [comment]
Even with custom roles it is not possible (that I can see) to grant "[Transfer issues](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role)" to [a custom role](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-peoples-access-to-your-organization-with-roles/managing-custom-repository-roles-for-an-organization#issue). :-/

**Reply by Chew** [reply]
I also checked and there's no way to let them edit issues like I wanted either. Custom roles is upsetting :(

**Reply by ccolella-mdc** [reply]
The wost part is you can't even make a custom role and grant the ability to assign issues to projects. The only option is to give people the "Write" role

**Comment by andreas-venturini** [comment]
> Assign to Projects (not just Milestones)

A new team member of ours (w/ triage rights to the repo) just asked me why they couldn't add an issue to a project - but to a milestone. Before seeing this thread here I thought this was a bug. I had to give them write rights to fix this (which isn't clear from the description of the roles at all)

![image](https://user-images.githubusercontent.com/4801917/169303424-ecb56c22-2bdc-4cca-aeb7-b80108de367b.png)

**Comment by davestewart** [comment]
We have a similar issues.

We want to give our PMs free rein to manage the direction of the product on a day-to-day level, but also want to shield them from the technical side.

Is there any way to do this?

**Comment by mubshrx** [comment]
Having the same problem. I want to give some people permission to create and manage issues but not write access.

**Comment by mattab** [comment]
We also really need ability for "Triage" permission to let people fully manage issues, **rename issue titles & moderate issues.**

**Comment by jensf** [comment]
## "Triage" role should allow **checking off todo items** 

Like this one:
- [ ] confirm with client mockup is ok _(pm cannot complete this task)_
- [ ] UAT completed _(client agent cannot complete this task)_
- [ ] assets available on onedrive share _(graphics designer cannot complete this task)_

I though the `triage` role was perfect for these types of employees who won't be committing any code, but still contributing to the overall project.

**Comment by sylver** [comment]
`Triage` role is basically useless and does not provide the permissions expected by its description.

I don't think that would be really hard to adapt those permissions to the intended usage, so why are we still forced to use the `Write` permission when we want someone to only manage issues (and tasks) properly ?

**Comment by schammah** [comment]
Creating milestones is also missing for `Triage` Role

**Comment by johnshreflermellon** [comment]
I echo @schammah's request; creating milestones is limited to the Write role. This should be a Triage permission.

**Comment by harshita-srivastava-yral** [comment]
I as PM would not want the write access to complete repos but definitely want write and edit the issues created. As a triage, I can't even edit an issue. This looks like a major bug on Github end. Awaiting resolution.

**Comment by zachleat** [comment]
Just received a report that the Triage role can’t rename issue titles, which seems like a bug to me? Can that be changed?
